examples/tutorials/analysis-1d/spectral_analysis_rad_max.py

```diff
@@ -349,4 +349,32 @@
 ax.set_ylim([1e-13, 1e-10])
 plt.show()
 
+
+######################################################################
+# Dataset simulations
+# -------------------
+#
+# A common way to check is a fit is biased is to simulate multiple datasets with
+# the obtained best fit model, and check the distribution of the fitted parameters.
+# Here, we show how to perform one such simulation
+#
+
+dataset_simulated = datasets.stack_reduce().copy(name="simulated_ds")
+simulated_model = best_fit_model.copy(name="simulated")
+dataset_simulated.models = simulated_model
+dataset_simulated.fake(
+    npred_background=dataset_simulated.counts_off * dataset_simulated.alpha
```

Comment:
This line ignores the data from the in measurement. The predicted background counts in the Wstat case depend on the predict counts for the source. Instead they are given by https://docs.gammapy.org/1.0.1/api/gammapy.stats.get_wstat_mu_bkg.html#gammapy.stats.get_wstat_mu_bkg

However this is only valid for a given source model! We should at least mention this assumption...